app/controllers/placements/schools/placements_controller.rb

```diff
@@ -22,9 +22,20 @@ def edit_mentors
   end
 
   def update
-    @placement.update!(placement_params)
+    unless valid_path?
+      redirect_to placements_school_placement_path(@school, @placement),
+                  flash: { alert: t("errors.internal_server_error.page_title") } and return
```

Comment:
Are we sure we want to show "Sorry, there’s a problem with the service"?

Reply:
The only way this happens is if a user deliberately changes the URL in the form via the inspector. I think the message is fair given the circumstances